app/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
@the-dagger Thanks for pointing out, I will remove all keys from the repo and will send the separate PR using firebase JSON.